pennylane/ops/channel.py

```diff
@@ -365,6 +365,92 @@ def _kraus_matrices(cls, *params):
         return [K0, K1, K2, K3, K4]
 
 
+class PauliError(Channel):
+    r"""PauliError(operators, p, wirelist, wires)
+    Arbitrary number qubit, arbitrary Pauli error channel.
+
+    This class models corresponding Kraus matrices to a series of arbitrary Pauli operators on
+    an arbitrary amount of wires. The operators are supplied in a string. The according wires are
+    supplied in an array called wirelist. The probability with which the channel is applied is
+    supplied as a float. Each wire can take one Pauli operator.
+
+    This channel is modelled by the following Kraus matrices:
+
+    .. math::
+        K_0 = \sqrt{1-p} * I
+
+    .. math::
+        K_1 = \sqrt{p} * (K_{w0} \otimes (K_{w1} \otimes (\dots K_{w_{max}})))
```

Comment:
Minor: would we need to start a bracket after each `\otimes`?

Reply:
I think so, because the Kronecker Product is non-commutative, right?

Reply:
nevermind, you're right 🤦‍♂️

Reply:
You're right, we don't